### ISSUE TITLE
REFPROP upgrade from v9.X to v10 - Are any fluids depreciated, if so which ones?

### ISSUE THREAD

**Issue by nathan-kamphuis** [usnistgov/REFPROP-wrappers#250]
Hi REFPROP team,

I want to say thanks for the amazing work you all have done.

Six months ago when RP v10 was installed, it was noticed that the fluid list was much shorter than RP v9. I assumed that the team simply hasn't had a chance to upgrade all the fluid files yet. Some of the missing fluids are older refrigerants that are no longer used by industry, yet may be used in research. Would the team share what fluids are currently missing and which ones they intend to depreciate for RP v10.

Nate

If you have an issue to report, please continue and fill out the applicable sections below.  The details provided will help to resolve this issue as quickly as possible.

### Prerequisites

**NOTE**: Be aware that all issues are publicly accessible and viewable.  Thus please do not post any code or other content that is protected intellectual property or under copyright.

Before posting an issue, please:  
    * Check the [FAQ](https://pages.nist.gov/REFPROP-docs/) for common solutions.  
    * Check that your issue isn't already filed under [REFPROP Issues](https://github.com/usnistgov/REFPROP-issues/).  
    * Check the [documentation](http://refprop-docs.readthedocs.io/en/latest/).  

You can also use markdown to format your issue: [GitHub guide on Markdown](https://guides.github.com/features/mastering-markdown/). If you have code snippets, please use a code block to ensure that the formatting remains legible in the web interface.  For instance, surround your code in triple backticks:

``` python
print('Hello world!')
```

### Description

[Description of the issue]

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

**Expected behavior:** [What you expect to happen]

**Actual behavior:** [What actually happens]

### Versions

**REFPROP Version:** [REFPROP version you are using]  
**Operating System and Version:** [OS you are running REFPROP on]  
**Access Method:** [How you are accessing REFPROP (i.e. Python, other wrapper)]  

### Additional Information

If possible, please post examples and/or screenshots of the issue.

**Comment by ianhbell** [comment]
Nope, did you try the All Fluids radio option in the fluid selection?  That will turn on all fluid files that can be found.

**Comment by ianhbell** [comment]
Please also see https://github.com/usnistgov/REFPROP-issues/issues/255
